rust/relay/src/main.rs

```diff
@@ -178,7 +178,7 @@ fn setup_tracing(args: &Args) -> Result<()> {
         &tracing_subscriber::registry().with(log_layer(args)).into(),
     );
 
-    let dispatch: Dispatch = match args.otlp_grpc_endpoint {
+    let dispatch: Dispatch = match args.otlp_grpc_endpoint.clone() {
         None => tracing_subscriber::registry().with(log_layer(args)).into(),
         Some(endpoint) => {
             let grpc_endpoint = format!("http://{endpoint}");
```

Comment:
HTTP? Does gRPC upgrade to TLS internally or something?

Reply:
Yes, gRPC runs on top of HTTP2 over TLS.